Tools/Plotfile/fcompare.cpp

```diff
@@ -113,10 +117,15 @@ int main_main()
     AMREX_ALWAYS_ASSERT_WITH_MESSAGE(pf_a.spaceDim() == pf_b.spaceDim(),
                                      "ERROR: plotfiles have different numbers of spatial dimensions");
 
-    const int finest_level = pf_a.finestLevel();
+    const int finest_level = std::min(pf_a.finestLevel(), pf_b.finestLevel());
     const int nlevels = finest_level+1;
-    AMREX_ALWAYS_ASSERT_WITH_MESSAGE(pf_a.finestLevel() == pf_b.finestLevel(),
-                                     "ERROR: number of levels do not match");
+    if (allow_diff_num_levels) {
+        amrex::Print() << "\n WARNING: number of levels do not match\n";
```

Comment:
Should there be a test comparing the number of levels before printing the message?